x/nexus/keeper/msg_server.go

```diff
@@ -56,12 +56,12 @@ func (s msgServer) RegisterChainMaintainer(c context.Context, req *types.Registe
 	for _, chainStr := range req.Chains {
 		chain, ok := s.GetChain(ctx, chainStr)
 		if !ok {
-			s.Logger(ctx).Error(fmt.Sprintf("%s is not a registered chain", chainStr))
+			s.Logger(ctx).Debug(fmt.Sprintf("%s is not a registered chain", chainStr))
```

Comment:
may want to add some more context for this logging msg, so we know what the cause, e.g. the other log contains ` skipping maintainer registration`

Reply:
should return an error instead, but that can be a separate commit (so we can include this as a patch).

Reply:
changing it to returning an error is a breaking change and therefore cannot be a patch

Reply:
yeah i meant that we should have it return an error on main but change the log level on the release branch for the patch.